### PR TITLE
README updates for building

### PR DESCRIPTION
**Pull request by birdsarah**
Improvements to README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -335,14 +340,17 @@ APP_ID_IOS = org.mozilla.ios.FirefoxVPN
...
+Then generate the Xcode project using our script (and an optional adjust token):
+
 ```bash
-./scripts/macos/apple_compile.sh ios [--adjust <adjust_token>]
+./scripts/macos/apple_compile.sh ios [--adjust <adjust_token>] -q ~/Qt/6.2.4/macos/bin
 ```
-(If you get an error like `Step 7: Generate translation resources...
-sh: /Users/[username]/Qt/6.2.4/ios/bin/lconvert: No such file or directory`
-you may need to provide the macOS Qt bin folder through the -q flag:
-`./scripts/macos/apple_compile.sh ios -q ~/Qt/6.2.4/macos/bin`)
```
Comment (birdsarah) [inline README.md:345]: Seems easier just to tell people to use the -q flag.

File: README.md [hunk 2]
```diff
@@ -406,7 +414,7 @@ it from the [official website](https://golang.org/dl/).
 3. Set the `QT_HOST_PATH` environment variable to point to the location of the `androiddeployqt` tool.
 
 4. Set the `ANDROID_SDK_ROOT` and `ANDROID_NDK_ROOT` (required version: 21.0.6113669) environment variables,
-to point to the Android SDK and NDK intallation directories.
```
Comment (birdsarah) [inline README.md:409]: typo

File: README.md [hunk 3]
```diff
@@ -260,6 +261,9 @@ time.
 
 #### Building with Xcode
 
+Before you start this process, if you have a fresh install of Xcode, open it and sign-in.
```
Comment (mcleinman) [inline README.md:264]: Can we make this a bit more straightforward for people who might not know Xcode. Something like a well-worded "Open Xcode, go to settings, accounts, sign in w/ your Apple ID."

File: README.md (hunk 1 above)
Comment (mcleinman) [reply README.md:345]: 👍🏻

File: README.md (hunk 2 above)
Comment (mcleinman) [reply README.md:409]: Thank you!

**Review by mcleinman — COMMENTED** [review]
Thanks for this, very helpful. One small request for some additional clarity for people new to Xcode.

**Review by Gela — APPROVED** [review]

**Comment by mcleinman** [comment]
I'm assuming failing tests aren't due to readme change. @birdsarah , should I go ahead and merge anyway? Or wait for tests to be fixed, for good practice?

**Pull request merged**